.default_env

```diff
@@ -1,2 +0,0 @@
-SLACK_AUTH_TOKEN=
-SLACK_SIGNING_SECRET=
```

Comment:
Why deleting this? Those are so people can do initial configuration easily by just copying it to .env file and sourcing it.